backend/handlers.py

```diff
@@ -61,7 +61,14 @@ def write_error(self, status_code, **kwargs):
         """ Overwrites write_error method to have custom error pages.
         http://tornado.readthedocs.org/en/latest/web.html#tornado.web.RequestHandler.write_error
         """
-        logging.info("Error do something here again")
+        level = kwargs.get("level", "error")
+        msg = kwargs.get("msg", "An unspecified error occurred.")
+        if level == 'error':
+            logging.error(msg)
+        if level == 'warning':
+            logging.warning(msg)
+        if level == 'info':
+            logging.info(msg)
```

Comment:
A default case here in case `level` is something else.

As a further note, this method is still not doing what it should do (it should basically end with a `self.finish`). We can take that later, but maybe add a TODO about the need to fix this further.